test/service-env.test.ts

```
@@ -419,7 +418,7 @@ describe("service environment", () => {
             .replaceAll("/tmp/nemoclaw-proxy-env.sh", `${fakeDataDir}/proxy-env.sh`),
         ].join("\n");
         writeFileSync(tmpFile, wrapper, { mode: 0o700 });
-        const runOpts = { encoding: /** @type {const} */ "utf-8" };
+        const runOpts: { encoding: BufferEncoding } = { encoding: "utf-8" };
```

ESLint

[error] 'BufferEncoding' is not defined.

(no-undef)